spec/_layouts/default.html

```diff
@@ -80,18 +80,18 @@
 					<a class="navbar-brand hidden" href="#">Navbar</a>
 					<div class="nav-collapse collapse navbar-inverse-collapse">
 						<ul class="nav navbar-nav">
-              <li><a href="{{base}}index.html">Home</a></li>
+              <li><a class="nav-link" href="{{base}}index.html">Home</a></li>
                 <li class="dropdown">
                   <a href="#" data-toggle="dropdown" class="dropdown-toggle">Conformance<b class="caret">
                   </b>
                 </a>
                 <ul class="dropdown-menu">
-                  <li><a href="{{base}}conformance/index.html">Declaring Conformance</a></li>
-                  <li><a href="{{base}}conformance/core-set/index.html">Conformance: Core Set</a></li>
-                  <li><a href="{{base}}conformance/index.html">Conformance: Capabilities Sets</a></li>
-                  <li><a href="{{base}}capability-statement/index.html">Syntax for declaring Conformance</a></li>
+                  <li><a class="nav-link" href="{{base}}conformance/index.html">Declaring Conformance</a></li>
+                  <li><a class="nav-link" href="{{base}}conformance/capability-sets/index.html">Conformance: Capabilities Sets</a></li>
```

Comment:
Looks like this is a longstanding typo: should say "Capability Sets" rather than "Capabilities Sets".